news/2020-11-19-better-protecting-players-from-abuse.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Better protecting players from abuse"
+date: 2020-11-19 09:00:00 +0000
+---
+
+Hi everyone, peppy here.
+
+osu! to me has always been about freedom of speech and action. I've always strived to continue running the service with the minimum possible level of moderation, intervention and imposition on the players and the team.
+
+In the last week, we have had some players come forward and describe events that happened between themselves and prominent community members (often in voluntary team positions like the Global Moderators or Beatmap Approval Team) which can objectively be defined as "wrong" from a moral standpoint, involving inappropriate exchanges where one party is under the age of 18.
```

Comment:
```suggestion
In the last week, we have had some players come forward and describe events that happened between themselves and prominent community members (often in voluntary team positions like the Global Moderators or Beatmap Nominators) which can objectively be defined as "wrong" from a moral standpoint, involving inappropriate exchanges where one party is under the age of 18.
```

Reply:
hmm, it was actually from an era when they were called BAT (yeah, that long ago)

Reply:
one case involves BAT era stuff so yeah, this doesn't need renaming, no cases handled recently involve beatmap nominators iirc